vision/use-cases/Advanced_prompting_for_image_generation_on_Imagen2.ipynb

```diff
@@ -0,0 +1,651 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #39.        if disable_enhanced_prompting is False:
</pre><p>In general, boolean names should always be positive, makes code more readable. Change this variable to <code>enable_enhanced_prompting</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/549/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>